java/org/apache/catalina/startup/Tomcat.java

```diff
@@ -1290,9 +1290,9 @@ public static void main(String[] args) throws Exception {
         // Process some command line parameters
         String[] catalinaArguments = null;
         for (int i = 0; i < args.length; i++) {
-            if (args[i].equals("--no-jmx")) {
+            if ("--no-jmx".equals(args[i])) {
```

Comment:
The change to equals() is subjective. I'd argue the current code is easier to read.